setup.py

```diff
@@ -30,12 +24,11 @@
     author='Divio AG',
     author_email='info@divio.ch',
     url='https://github.com/divio/aldryn-style',
-    packages=['aldryn_style', 'aldryn_style.migrations'],
-    install_requires=INSTALL_REQUIRES,
+    packages=find_packages(),
     license='LICENSE.txt',
     platforms=['OS Independent'],
+    install_requires=REQUIREMENTS,
     classifiers=CLASSIFIERS,
-    long_description=open('README.md').read(),
```

Comment:
Definitely want to keep these items, @FinalAngel and looks like we need to add them to NewBlog too.